gnomad_constraint/resources/resource_utils.py

```diff
@@ -22,9 +22,16 @@
 VERSIONS = ["2.1.1"]
 CURRENT_VERSION = "2.1.1"
 DATA_TYPES = ["context", "exomes", "genomes"]
+MODEL_TYPES = ["plateau_models", "coverage_model"]
```

Comment:
```suggestion
MODEL_TYPES = ["plateau", "coverage"]
```
this feels just as clear and is more succinct